pkg/client/client.go

```diff
@@ -6,12 +6,15 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hewlettpackard/hpegl-provider-lib/pkg/client"
+
 	api_client "github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/client"
+
```

Comment:
Please fix the ci issues

Reply:
I've added unit-tests to the latest commit to that repo.